.github/workflows/wheels.yml

```diff
@@ -190,17 +190,30 @@ jobs:
 
     - run: make sdist
 
-    - uses: actions/upload-artifact@v3
+    - uses: actions/upload-artifact@v4
       with:
-        name: dist
+        name: dist-sdist
         path: dist/*.tar.gz
 
-  success:
-    permissions:
-      contents: none
+  pypi-publish:
+    # TEMP for testing
+    # if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    if: github.event_name == 'push'
```

Comment:
We're actually getting a PR event here. 
```suggestion
```

Reply:
I'm not sure you'd be able to publish from PRs.